core/domain/feedback_jobs_one_off_test.py

```diff
@@ -88,7 +88,7 @@ def _run_one_off_job(self):
 
     def test_message_count(self):
         """Test if the job returns the correct message count."""
-        with self.swap(feconf, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', False):
+        with self.swap(constants, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', False):
```

Comment:
One off jobs are all mapped to FeedbackThreadModel (the old one). So all of these tests will work only on the old framework.

Reply:
So what happens when we switch to the new framework and run any of these one-off jobs?

Also if these are going to be deprecated after the flag change, then they should go in an old_feedback_one_off_jobs_test.py file, just like all the rest.

Reply:
On running after switching, It will just run on entities of the old model, and the changes will not reflect in the newly created models. The jobs here also include jobs to convert from old models to new models. I think these tests are not applicable after the flip, and need to be deprecated after.

Reply:
I see. The codebase needs notes to that effect and there needs to be a TODO recording the deprecation tasks.

Reply:
(Sorry, not a TODO -- I meant a GitHub issue.)

Reply:
Hang on -- I looked at this again. The correct thing to have done here would have been to modify the job so that there's a "new" job and an "old" job. The handler would redirect to the correct one based on the status of the flag.

However, I don't think this job is actively being used, and it hasn't been run in a while. So -- in the interest of time, let's just delete it altogether. We can reimplement it later if needed.

Reply:
Ok.

Reply:
Renamed the test file.